containers/java-postgres/.devcontainer/docker-compose.yml

```diff
@@ -0,0 +1,58 @@
+version: '3.8'
+
+volumes:
+  postgres-data:
+
+services:
+  app:
+    container_name: javadev
+    build: 
+      context: .
+      dockerfile: Dockerfile
+      args:
+        # Update 'VARIANT' to pick an version of Java: 11, 17.
+        # Append -bullseye or -buster to pin to an OS version.
+        # Use -bullseye variants on local arm64/Apple Silicon.
+        VARIANT: 11-bullseye
+        # Options
+        INSTALL_MAVEN: "true"
```

Comment:
Isn't this "forcing" the container to install always MAVEN?

Reply:
It is, but we need maven for the tests to pass. There's no other way to download JDBC right?

Reply:
upon additional testing, the `test.sh` configures maven first before running the tests, so @juzuluag is correct that we don't need to require maven earlier